trollmoves/movers.py

```diff
@@ -23,6 +23,7 @@
 
 """Movers for the move_it scripts."""
 
+from doctest import ELLIPSIS_MARKER
```

Comment:
I don't see this in `main` at all, so don't know where it's coming from. Should be safe to remove.

Reply:
Oh, actually, it's coming from this PR since it shows up in the diff.